### PR TITLE
Marian 774 styleguidepage

### PR DESCRIPTION
**Pull request by mmlumba**
References #774

First iteration of style guide yay :)

Base page:
screen shot 2017-02-13 at 5 35 30 pm

With expanded accordion and code snippet:
screen shot 2017-02-13 at 3 19 38 pm

**Notes:**
UI revisions to the style guide will be made later, but for now this should cover the following acceptance criteria:
* When a system admin navigates to /admin/styleguide they should see a page with the heading "Styleguide"
* There should be a brief description of the purpose of the styleguide here.
* The first component documented should be the modal. There should be an example button that opens an example modal. There should also be a code snippet of how to make the associated modal.

### CODE REVIEW

**Comment by Chingujo** [comment]
Why so amazing @mmlumba?!

A intro:

Caseflow Commons is home to our most up to date style guide, UI Kit, and code for Caseflow products. The goal is to keep consistent stylings across Caseflow applications to keep the interface predictable and familiar to the user. The consistency also helps us reuse common code across our apps and increase design efficiency through standardized stylings.

File: app/views/admin/styleguide/_modals.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+&lt;%<br />
```
Comment (shanear) [inline app/views/admin/styleguide/_modals.html.erb]: woof it's gonna be really hard to update this code with all the `&lt;` and `&gt;`

We should look for a better way to generate the markup. But add that as a tech improvement story.

**Comment by mmlumba** [comment]
Also @KHarshawat @Chingujo is there a commons logo for the style guide I should be using?

**Comment by lakohl** [comment]
Great job @Chingujo ! Below is a version with a couple tiny tweaks to consider.

Caseflow Commons is home to our most up to date style guide, UI Kit, and code for Caseflow products. The goal is to maintain consistent styling across Caseflow applications and to keep our interface predictable and familiar to the user. This unified system also helps us reuse common code across our apps and increase the efficiency of the design process. 

Also, lol we should decide on how we are writing style guide/StyleGuide/styleguide. Personally, I like the way you have it in the intro.

File: app/views/admin/styleguide/_modals.html.erb (hunk 1 above)
Comment (mmlumba) [inline app/views/admin/styleguide/_modals.html.erb]: @shanear : I was looking at 2 options for syntax highlighting that leave the original markup in-tact. I'll make a general "styleguide improvement" ticket and add this comment in the ticket:
* Rouge (ruby gem): http://rouge.jneen.net/
* Highlight.js (npm package): https://highlightjs.org/

Comment (shanear) [inline app/views/admin/styleguide/_modals.html.erb]: Cool link me to the ticket and look at all the options and pitch which one you want to use and why.

**Comment by mmlumba** [comment]
@KHarshawat-gov views for commons with the white background:
screen shot 2017-02-14 at 10 35 06 am
screen shot 2017-02-14 at 10 35 55 am

**Pull request merged**